CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.1.2
+  - Adds "Technology Preview" call-out to documentation [#4](https://github.com/logstash-plugins/logstash-input-elastic_serverless_forwarder/pull/4)
```

Comment:
```suggestion
  - [DOC] Adds "Technology Preview" call-out to documentation [#4](https://github.com/logstash-plugins/logstash-input-elastic_serverless_forwarder/pull/4)
```

Reply:
Please add Doc label so that we won't have to make a manual fix when we generate release notes